.github/workflows/integration-test.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         with:
           path: OpenSearch-Dashboards
           repository: opensearch-project/OpenSearch-Dashboards
-          ref: 'main'
+          ref: '2.0'
```

Comment:
Shouldn't we fix this issue separately from this backport?

Reply:
Yes, we should.